hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/CatyGZ/hw-golang-otus
```

```suggestion
module github.com/CatyGZ/hw-golang-otus/hw01_hello_otus
```
Рекомендую указать полный путь до модуля, чтобы он был доступен для скачивания